libs/wingsdk/.projen/tasks.json

```diff
@@ -345,6 +345,15 @@
         }
       ]
     },
+    "pre-compile-force": {
```

Comment:
imo I think the existing `pre-compile` task should just always run with --force.

Since we use `turbo`, you generally will not run this more than once anyways

Reply:
Well, I did not really take into account that it is only used once, and I apparently over-thinked this. I will change it right away, thank you :)